addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -83,6 +83,14 @@ def _constrains_l10n_id_tax_number(self):
             elif record.l10n_id_tax_number[2] not in ('0', '1'):
                 raise UserError(_('The third digit of a tax number must be 0 or 1'))
 
+    @api.model_create_multi
```

Comment:
@pivi-odoo default value or compute field would not work?  

And what happens in the pos when you create a new partner?  It will have a correct default value or should you ideally be able to change it in the pos?

Reply:
I've tested both cases:
- using a default value doesn't work, because the self passed to the function is an empty account.move recordset, and I cannot reference the partner_id, and I don't have access to the raw partner_id to query with env.
- using a compute you do get a populated self with the account.move, but since the field is readonly and is not stored, the ORM denies the assignation. Setting the field to writable doesn't seem to set the value correctly either.

In the targeted version (14.0), the "Create Customer" in POS is really rudimentary, there is no tab/view to add any of the tax related information, so no, there are no default, the user needs to setup the customer details in the Contacts app before invoicing them through POS.

Reply:
Just tested, the ORM throws a fit at https://github.com/odoo/odoo/blob/14.0/odoo/fields.py#L1028

Reply:
@pivi-odoo That error is because somehow the value is not being set in the compute method.  Maybe you can share me that code, so I can check and try as well.

Reply:
I've managed to get it to work, I've pushed the version with a compute, seems to work as intended now.